pkg/core/bootstrap/autoconfig.go

```diff
@@ -223,3 +229,18 @@ func saveKeyPair(pair tls.KeyPair, dir workDir) (string, string, error) {
 
 	return crtFile.Name(), keyFile.Name(), nil
 }
+
+func autoconfigureInterCp(cfg *kuma_cp.Config) error {
+	if cfg.InterCp.Catalog.InstanceAddress != "" {
+		return nil
+	}
+	ips, err := util_net.GetAllIPs(util_net.NonLoopback)
+	if err != nil {
+		return errors.Wrap(err, "could not list all IPs of the machine")
+	}
+	if len(ips) == 0 {
+		return errors.New("there is 0 non-loopback interfaces on the machine. Set KUMA_INTER_CP_CATALOG_INSTANCE_ADDRESS explicitly.")
+	}
```

Comment:
Hmm, maybe we could log a warning when there would be more than 1 non-loopback interface?

Reply:
good idea. Added a log